pkgs/by-name/so/sobjectizer/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  callPackage,
+  buildExamples ? false,
+  buildTests ? false,
```

Comment:
Under what circumstance would you not want to build the tests?

Reply:
In my vision the tests building is on demand.
On the other hands, if someone no need the tests, so he/she can disable it explicitly.

Should we always build the tests?

Reply:
> Under what circumstance would you not want to build the tests?

In my vision the tests building is on demand.
On the other hands, if someone no need the tests, so he/she can disable it explicitly.

Should we always build the tests?